tests/test_nsga2_multi_objective.py

```diff
@@ -43,6 +50,9 @@ def setup_optProb(self, n_obj):
         if n_obj == 2:
             self.optProb.addObj("obj2")
 
+    @unittest.skipIf(
```

Comment:
I am sorry to have checked this too late @nwu63 , but I don't really get the point of this step here. Can't we remove this altogether?  We are already skipping all `n_obj == 2` tests by default on all platforms. Is this related to an additional test that `testflo` runs on windows?
Locally I only have the last two tests of this stack:
```
about to run test_nsga2_multi_objective.py:TestNSGA2.test_opt .\test_nsga2_multi_objective.py:TestNSGA2.test_opt  ... SKIP (00:00:0.00, 0 MB)
test_nsga2_multi_objective.py fails on windows with two objectives! Skipping for now.
    about to run test_nsga2_multi_objective.py:TestNSGA2.test_opt_0 .\test_nsga2_multi_objective.py:TestNSGA2.test_opt_0 ... OK (00:00:4.17, 0 MB)
    about to run test_nsga2_multi_objective.py:TestNSGA2.test_opt_1 .\test_nsga2_multi_objective.py:TestNSGA2.test_opt_1  ... SKIP (00:00:0.00, 0 MB)
skip flaky NSGA2 tests
```

Reply:
This skips all tests on Windows, the below skips all multiobjective tests, so the only tests being run are the single-objective tests on linux.

Reply:
I understand the idea (and that is what I was expecting) but that printout in my comment is from the Windows test log - apparently the single-objective test is running on Windows? Or is it a Testflo fluke?

Reply:
It is possible that the skipIf doesn't play nicely with parameterized. _However_ this is actually what we want---the single objective test should pass on Windows and we should test it, while the multi-objective test appears to be flat out broken. Let me throw together a PR to update this.